go.mod

```diff
@@ -40,11 +40,11 @@ require (
 	google.golang.org/grpc v1.53.0
 	gopkg.in/yaml.v2 v2.4.0
 	helm.sh/helm/v3 v3.12.2
-	k8s.io/api v0.27.2
+	k8s.io/api v0.27.4
```

Comment:
This is not technically necessary, can revert if required.

Reply:
Since everything else is at 0.27.2, I'd suggest reverting.

Reply:
If possible, I'd suggest reverting everything back to 0.27.2, but I realize that may not always be possible.
Having different versions of the k8s modules across the api/operator-registry/operator-lifecycle-manager repos will cause us headaches when downstreaming.